CONTRIBUTING.md

```diff
@@ -0,0 +1,52 @@
+# Contributing
+
+### About
+
+FBX is a very widely used and flexible file format.
+We currently expect most models **to not load properly**.
+We cannot test how `bevy_mod_fbx` handles the export formats of every software out there.
+If your model loads properly, thank your lucky start,
+if you encounter any issue please do the following:
+
+- Try opening the model using the `scene_viewer` (run `cargo run --example scene_viewer --release -- /path/to/file.fbx`)
+  (if your file has textures, make sure to enable the correct file formats using `--features bevy/png bevy/jpg bevy/tga` etc.)
+- If it fails, open an [issue] on our Github repo with a screenshot in the scene viewer
+  and a screenshot of how the model should look like
+- Ideally provide a download link to your FBX model
```

Comment:
Probably worth adding a issue template as well <https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository>

Reply:
I think so